src/wallet/mineTransaction.ts

```diff
@@ -67,6 +67,9 @@ class MineTransaction extends EventEmitter {
 
     // Promise that resolves when push tx finishes with success
     // or rejects in case of an error
+    // TODO: If this promise is not handled properly we crash the client application with a
```

Comment:
Do we have an issue to track this?

Reply:
That's a good idea. Opened 642 to track it:
- https://github.com/HathorNetwork/hathor-wallet-lib/issues/642